test/webhook.e2e-spec.ts

```diff
@@ -8,23 +8,23 @@ import { User } from '@prisma/client';
 import { HttpModule } from '@nestjs/axios';
 import { WebhookService } from '../src/webhook/webhook.service';
 import { WebhookRepository } from '../src/webhook/webhook.repository';
-import { AuthService } from '../src/auth/auth.service';
 import {
   createWebhookRequest,
   createWebhookWithUserId,
 } from '../src/webhook/test/webhook.factory';
 import { WebhookModule } from '../src/webhook/webhook.module';
 import { CryptoUtils } from '../src/webhook/utils/crypt-webhook-token';
 import { AuthModule } from '../src/auth/auth.module';
+import { JwtService } from '@nestjs/jwt';
 
 describe('WebhookController (e2e)', () => {
   let app: INestApplication;
-  let authService: AuthService;
   let prismaService: PrismaService;
   let user: User;
   let accessToken: string;
+  let jwtService: JwtService;
 
-  beforeAll(async () => {
+  beforeEach(async () => {
```

Comment:
Why did you change it to `beforeEach`? I think testing module should be created once. Now you are creating it for each test.

Reply:
I've noticed that I did not have a call to database that deletes all webhooks and its events before. So to fix that, what I did is change it so it runs multiple testing modules. 

But I changed it, so test module is created once, and records created in test are being deleted after.